pylint/checkers/variables.py

```diff
@@ -2511,7 +2512,7 @@ class D(Tp):
             and name in frame_locals
         )
 
-    # pylint: disable = too-many-branches
+    # pylint: disable = too-many-branches, too-many-statements
```

Comment:
```suggestion
    # pylint: disable-next=too-many-branches,too-many-statements
```
The whitespace is up to you. I would just recommend to use `disable-next`.